### PR TITLE
JetpackGameStudios/NewUnityGame:master to afroraydude/NewUnityGame:master

### PR DESCRIPTION
**Pull request by afroraydude**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/afroraydude/NewUnityGame/pull/2" target="_blank">ReviewNinja

File: update.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+curl -o update.zip http://afroraydude.sytes.net/tradesecret/update.zip
```
Comment (afroraydude) [inline update.sh]: !completed

Comment (afroraydude) [inline update.sh]: !fix

Comment (afroraydude) [inline update.sh]: Line Note

Comment (afroraydude) [inline update.sh]: !fixed

**Pull request merged**